### DISCUSSION TITLE
Bloatynosy Nue 0.87

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#241] [Announcements]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Bloatynosy Nue! The king is back... The original repo has been revived and is now online <a href="https://t.co/iCiPTbYhlw">https://t.co/iCiPTbYhlw</a> <a href="https://t.co/qo8FbL9uTj">pic.twitter.com/qo8FbL9uTj</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1857164868954325081?ref_src=twsrc%5Etfw">November 14, 2024</a></blockquote>

- **The UI has been further simplified and modernized, while also drawing inspiration from the classic BloatyNosy app to maintain that same level of simplicity**
- Stable release on [version 0.85](https://github.com/builtbybel/Bloatynosy/releases/tag/0.85.0) with migrated features and full support for Windows 11 24H2, especially 23H2, but also older feature versions - **this was the version everyone agreed was the best of its time!**

![explorer_O2ajNTZiEi](https://github.com/user-attachments/assets/fe1a7af1-306f-4c75-9233-1a0b74884fec)

<details>
  <summary>If you haven’t heard yet, BloatyNosy has been brought back to life, read on to learn more...</summary>

You probably remember my classic BloatynosyApp, which had to go through some renaming and eventually got discontinued. Everything that came after, like the web-based "Winpilot," didn't last long either. Honestly, I thought I was done with Bloatynosy-well, at least with the name. But then, during a recent visit to a well-known company, I saw an employee Googling the app and trying to download it, and I had to stifle a laugh. Apparently, the legend lives on. The employee and I had a good laugh about it, and he even called it the epitome of debloating.
Sometimes, I think a little too long about things, and then a useful project just pops out. Bloatynosy was one of those projects. Sure, the name wasn't for everyone, but it was the go-to tool for cleaning up and debloating Windows. Recently, I added an OOBE assistant called "Fresh11" to the FlybyScript—basically, an all-in-one solution and I figured, why not use the same foundation to bring Bloatynosy back to life?

I had a hard time committing to it, but in the end, I went for it. Now I can proudly say that after Bloatynosy, there won’t be anything newer or better. This time, it'll be even leaner, more efficient, smaller, and faster. A simple .exe that anyone can understand, doing exactly what bugs us after installing Windows 11. **The reborn Bloatynosy carries the "Nue" extension (Bloatynosy Nue)**, not "New" ;) The last solid, classic version was 0.85, and that's where Im picking up (with a fresh new interface). The build carries a small 11.x version number.

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/0.87.100'>Bloatynosy Nue 0.87</a>.</em>

**Comment by G0ldheart** [comment]
Any chance we can get a dark mode?
